### PR TITLE
add of io repository

### PR DESCRIPTION
**Pull request by mracos**
Add of plugin for the io language

The build is failing on MacOS but i really don't know what is :(
And as i don't have a mac, i don't know how to debug it

### CODE REVIEW

**Comment by Stratus3D** [comment]
Sorry for the late review. It looks like the build for your plugin is failing (not the build for this repo - I'm aware it's not been working).

**Comment by mracos** [comment]
Sorry for the REALLY late response 😅
All fixed now

**Comment by Stratus3D** [comment]
Awesome! Thanks @mracos !

File: plugins/io [hunk 1]
```diff
@@ -0,0 +1 @@
+repository = https://github.com/mracos/asdf-io
```
Comment (Stratus3D) [inline plugins/io]: I think the actual Git repository name ends in `.git`. Sorry I just merged the PR, can you test this and create another one if this does not work?

Comment (mracos) [reply plugins/io]: Will look into it :smile:

**Pull request merged**